components/Dashboard/candidate/profile/update-skills-modal.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect, useState,KeyboardEvent } from 'react';
+import Modal from '@/components/modal/Modal';
+import { CancelIcon } from '@/assets';
+import { Button, OutlineBtn } from '@/components/button';
+interface modalcompProps {
+    open: boolean;
+    setOpen: any;
```

Comment:
This is a `React.SetState....` type, do not use any.

**Hint**: Hover over the where you declared the setOpen state and pick the correct type inference